### ISSUE TITLE
Basis creation fails at EBIT

### ISSUE THREAD

**Issue by joefowler** [usnistgov/Pope.jl#20]
@ggggggggg and Paul--does he have a github username?--note problems with estimation of ARMA models at the EBIT. The result is that basis creation fails or makes bad choices.

The branch `nonARMAnoise` has my attempt at a temporary fit for Julia 0.6. It uses the traditional MASS approach with the noise autocorrelation function, instead of the ARMA model.

Not tested yet, but we'll try tomorrow on EBIT data.

**Comment by joefowler** [comment]
Adding @PSzypryt

**Comment by joefowler** [comment]
Okay, @ggggggggg and @pszypryt, can we try the Julia0.6 code in branch `nonARMAnoise`? Use the new default method of "noisemass3". Here is how I did it on my computer. Replace `DIR` as appropriate for your data location.

I am happy to work with you to attempt this, but it works for me on 20190805 data using run 0000 for the noise and 0001 for the pulses. I was getting the `failedDomainError()making basis for channel 5` message on channels 5, 6, and 8. But with the updates, I am not.

```bash
DIR=/Volumes/Data2017/Data_EBIT
env -u PYTHONPATH julia-0.6 ./basis_create.jl --tsvd_method=noisemass3 --n_basis=4  --replaceoutput ${DIR}/20190805/0001/20190805_run0001_chan1.ljh  ${DIR}/20190805/0000/20190805_run0000_noise.hdf5
env -u PYTHONPATH julia-0.6 ./basis_plots.jl ${DIR}/20190805/0001/20190805_run0001_model.hdf5 
```

**Comment by joefowler** [comment]
Fixed by PR #22.
